plugins/shell-runner.koplugin/_meta.lua

```diff
@@ -0,0 +1,6 @@
+local _ = require("gettext")
+return {
+    name = "shellrunner",
+    fullname = _("Shell Runner"),
+    description = _([[Execute scripts instead of view them]]),
```

Comment:
So, which is better and which should be killed, between this as a plugin, and @NiLuJe 's "core" shell execution support ?

Reply:
the plugin is just to showcase the behaviour. It isn't intended to be merged (or even reviewed, outside the function that registers a handler)